sky/clouds/runpod.py

```diff
@@ -183,10 +181,16 @@ def make_deploy_resources_variables(
         else:
             custom_resources = None
 
+        if r.extract_docker_image() is not None:
+            image_id = r.extract_docker_image()
+        else:
+            image_id = r.image_id
```

Comment:
Let's use our kubernetes default image for runpod as well to speed up the provisioning:

```suggestion
        if r.image_id is None:
            # Use the same default image as kubernetes cloud for
            # the best performance.
            image_id = service_catalog.get_image_id_from_tag(
                'skypilot:gpu-ubuntu-2004', clouds='kubernetes')
        elif None in r.image_id:
            image_id = r.image_id[None]
        else:
            image_id = r.image_id[region.name]

        if image_id.startswith('docker:'):
            image_id = image_id[len('docker:'):]
```

Reply:
doesn't this duplicate some of the code from `extract_docker_image()` ?

Reply:
Hi @winglian, it is slightly different, as we have to `r.image_id` is a dict and we have to handle it. Also, we would like to set the default value for `image_id`. An alternative way is to:
```suggestion
            if r.image_id is None:
                image_id = 'runpod/base:0.0.2'
            elif r.extract_docker_image() is not None:
                image_id = r.extract_docker_image()
            else:
                image_id = r.image_id[r.region]
```